src/concrete/ml/quantization/quantized_module.py

```diff
@@ -170,6 +178,7 @@ def dump_dict(self) -> Dict:
         metadata["ordered_module_input_names"] = self.ordered_module_input_names
         metadata["ordered_module_output_names"] = self.ordered_module_output_names
         metadata["quant_layers_dict"] = self.quant_layers_dict
+        metadata["_onnx_preprocessing_str"] = self._onnx_preprocessing
```

Comment:
better if we keep the same name between the metadata dict and the class

Reply:
Yes done.